tests/configwizard_test.py

```diff
@@ -0,0 +1,292 @@
+import os
+
+import click
+import pytest
+
+from khal.configwizard import configwizard, choose_datetime_format, \
+    choose_time_format, create_config, create_vdir, \
+    get_vdirs_from_vdirsyncer_config, validate_int
+
+from khal.settings import get_config
+
+
+class InteractionStore():
+    """Used for mocking click.[confirm|prompt]"""
+    def __init__(self):
```

Comment:
Instead of doing this I would strongly recommend to end-to-end test the config wizard by running the entire CLI with `click.CliRunner` and comparing the output config.